src/adapter/http/src/data/router.rs

```diff
@@ -29,6 +29,18 @@ pub fn root_router() -> axum::Router {
             "/verify",
             axum::routing::post(super::verify_handler::verify_handler),
         )
+        .route(
+            "/workspace/info",
+            axum::routing::get(super::workspace_info_handler::workspace_info_handler),
+        )
+        .route(
+            "/me",
```

Comment:
perhaps `/accounts/me` is better?

Note: I'm preparing a proposal to improve endpoint schemes, so we will revisit soon.